compiler/qsc_frontend/src/error.rs

```diff
@@ -24,15 +21,17 @@ impl<E: Diagnostic> WithSource<E> {
     pub fn error(&self) -> &E {
         &self.error
     }
+
+    pub fn into_error(self) -> E {
+        self.error
+    }
 }
 
 impl<E: Diagnostic> WithSource<E> {
     /// Construct a diagnostic with source information from a source map.
     /// Since errors may contain labeled spans from any source file in the
     /// compilation, the entire source map is needed to resolve offsets.
     /// # Panics
```

Comment:
niptick: Looks like the panic text was removed but not the `# Panic` part of the doc comment. That can be dropped now too.

Reply:
It panics! What more do you wanna know? 😆